tools/StaticAnalysis/Exceptions/Az.IotHub/MissingAssemblies.csv

```diff
@@ -1,3 +1,4 @@
 "Directory","Assembly Name","Assembly Version","Referencing Assembly","Severity","ProblemId","Description","Remediation"
 "Az.IotHub","Microsoft.Win32.Registry","4.1.1.0","System.Management.Automation","0","3000","Missing assembly Microsoft.Win32.Registry referenced from System.Management.Automation","Ensure that the assembly is included in the Wix file or directory"
 "Az.IotHub","Microsoft.PowerShell.CoreCLR.Eventing","6.1.0.0","System.Management.Automation","0","3000","Missing assembly Microsoft.PowerShell.CoreCLR.Eventing referenced from System.Management.Automation","Ensure that the assembly is included in the Wix file or directory"
+"Az.IotHub","System.Reflection.TypeExtensions","4.0.0.0","Microsoft.Azure.Amqp","0","3000","Missing assembly System.Reflection.TypeExtensions referenced from Microsoft.Azure.Amqp","Ensure that the assembly is included in the Wix file or directory"
```

Comment:
'Microsoft.Azure.Amqp.dll' reference was removed, is this still needed?

Reply:
It is required otherwise giving error while static analysis checks.
![image](https://user-images.githubusercontent.com/31940305/73782107-522a2d00-4746-11ea-9c39-ca7b045bd3dc.png)